src/scripts/HorizontalDynseqTrack.js

```diff
@@ -197,7 +207,15 @@ export default function HDT(HGC, ...args) {
       }
       if (seqContainer.alpha) {
         for (let i = 0; i < sequence.length; i++) {
-          const charInd = (sequence.charCodeAt(i) & 95) - 65;
+          let letter = sequence[i];
+
+          if (this.options && this.options.reverseComplement) {
+            if (revComps[letter]) {
+              letter = revComps[letter];
```

Comment:
> Could you factor the first condition out of the loop and use firstCondition && revComps[letter] instead? Mostly for stylistic improvement.

I think factoring out the first condition would make the code more complicated for little benefit. Now you'd have to look in two places to understand what's happening. But I did get rid of the inner if statement and included in the first using `&&`.

> Also you could throw an error if `nonStandardSequence` and reverseComplement are selected at the same time, since they're pretty much mutually exclusive, and thereby you wouldn't need to check if the complement exists

What does `nonStandardSequence` do?

Reply:
Cool, I added `N` and `n` to the letter map.